model/AI/Properties/energyConsumption.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# energyConsumption
+
+## Summary
+
+Field captures the amount of energy consumed to build this AI package 
+
+## Description
+
+Text field that captures the energy consumption during training and operation of the AI model. a.k.a training energy consumption, inference energy consumption
+
+## Metadata
+
+- name: energyConsumption
+- Nature: DataProperty
+- Range: xsd:string
```

Comment:
The range of values of values is still emerging, so we're in the stage of collecting data.   It will be refined into a more explict unit in a future release.    This was considered, but from the examples looked at, we're not seeing enough standardization.

Reply:
Is the variability in the value or in what the energy is describing?  If the former, maybe you could use a key/value map that would allow more machine parsing but still some variability.